src/borg/xattr.py

```diff
@@ -68,7 +68,7 @@ def get_all(path, follow_symlinks=True):
         libc_name = 'libc.dylib'
     else:
         msg = "Can't find C library. No fallback known. Try installing ldconfig, gcc/cc or objdump."
-        logger.error(msg)
+        print(msg, file=sys.stderr)  # logger isn't initialized at this stage
```

Comment:
oh, I have missed that branch fully. 
I will have a look into it.
I was trying to get it working with mingw32 compiler ( not so straightforward way, but it is for fun).

Also thinking about using _paramiko_ as replacement of external ssh pipe.